TeadsSampleApp/Controllers/InRead/Mopub/ScrollView/InReadMopubScrollViewController.swift

```diff
@@ -7,7 +7,7 @@
 
 import TeadsSDK
 import UIKit
-import MoPub
+import MoPubSDK
```

Comment:
use conditional import to maximize compat with older mopub dependencies
```suggestion
#if canImport(MoPubSDK)
import MoPubSDK
#else
import MoPub
#endif
```